code/src/messages.ts

```diff
@@ -1,15 +1,18 @@
 import applicationModuleI18n from "@/shared/modules/application/i18n";
 import dashboardI18n from "@/pages/dashboard/i18n";
+import appPriductsListI18n from "@/layouts/app-products/i18n";
 
 const messages = {
   en: {
     ...applicationModuleI18n.en,
     ...dashboardI18n.en,
+    ...appPriductsListI18n.en,
```

Comment:
`Typo` here and also change it in other places

Reply:
Fixed